new/brewblox_service/service.py

```diff
@@ -117,8 +122,29 @@ def furnish(app: Type[web.Application]):
         )
     })
 
+    # Configure CORS and prefixes on all endpoints.
+    known_resources = set()
+    for route in list(app.router.routes()):
+        if route.resource in known_resources:
+            continue
+
+        # Add to known
+        known_resources.add(route.resource)
+
+        # Add prefix
+        route.resource.add_prefix(prefix)
+
+        # Add CORS
+        # TODO(Bob): Remove static resource instance check when aiohttp-cors bug is fixed
+        # Issue: https://github.com/aio-libs/aiohttp-cors/issues/155
+        if not isinstance(route.resource, web.StaticResource):
+            cors.add(route)
+            LOGGER.debug(f'Enabled CORS for {route.resource}')
+
     # Configure swagger settings
+    # We set prefix explicitly here
     aiohttp_swagger.setup_swagger(app,
+                                  swagger_url=prefix + '/api/doc',
                                   description='',
                                   title='Brewblox Service',
                                   api_version='0.3.0',
```

Comment:
Do we have a plan for api versioning per service? Should services override this? Or do we have a version for the service itself and for the import service library?

Reply:
Swagger api version should equal service implementation version. All our versioning - and linking to automatic deployment is todo.